package/scripts/ambari_helpers.py

```diff
@@ -43,7 +43,20 @@ def cdap_config(name=None):
        content=InlineTemplate(params.cdap_env_sh_template)
   )
 
-  cleanup_opts()
+  if name == 'auth':
```

Comment:
Is Cask standard for python two or four characters for indentation? Python standard is four...